core/templates/css/oppia.css

```diff
@@ -1733,7 +1733,7 @@ pre.oppia-pre-wrapped-text {
   font-family: "Roboto", Arial, sans-serif;
   font-size: 14px;
   margin-left: 5px;
-  margin-right: 5px;
+  margin-right: 0px;
```

Comment:
This change affects 3 other buttons, have checked if those are affected by this change?

Reply:
Yes, I just checked out, it does shift the two buttons to the right side just a bit. Also, it looks fine to me and isn't causing any problem in the website. 
Thanks

Reply:
Can you post screenshots of those as well.

Reply:
That's how it looks after the changes.

<img width="1440" alt="Screenshot 2021-02-02 at 12 48 49 PM" src="https://user-images.githubusercontent.com/47082288/106565699-33c36100-6555-11eb-8a55-9b54f6806efd.png">